GLAA.Services.Tests/Validation/When_validating_eligibility_criteria.cs

```diff
@@ -17,29 +15,34 @@ public void Setup()
             // valid model
             validModel = new EligibilityViewModel
```

Comment:
Should we rename this to `signupviewmodel` or something perhaps?

Reply:
Yeah, I wasn't sure if we were definitely calling this new process "Sign Up" but it's not like it's difficult to change